### ISSUE TITLE
Mapping of <C-;> or <C-S-;> not working

### ISSUE THREAD

**Issue by NGPONG** [neovide/neovide#1862]
**Describe the bug**
In my init.lua i have following keymap settings

```
map('n', '<C-;>', '<CMD>lua print(123)<CR>')
map('n', '<C-S-;>', '<CMD>lua print(123)<CR>')
map('n', '<C-:>', '<CMD>lua print(123)<CR>')
```

These keymaps work fine with neovim in windows terimal.

I know there is a problem with terminal emulators, they don't recognize control-shift chords. So usually I also have to wrote the following settings in windows terminal

```
    {
      "command": {
        "action": "sendInput",
        "input": "\u001b[58;5u"
      },
      "keys": "ctrl+shift+;"
    },
    {
      "command": {
        "action": "sendInput",
        "input": "\u001b[58;5u"
      },
      "keys": "ctrl+shift+;"
    },
```

But the problem is,  neovide is a GUI and not limited by terminal emulators, so I assume that neovide should support any keymap approach( Am i right? please correct me if i wrong XD ).

**To Reproduce**
Configure above mentioned mappings.

**Expected behavior**
Mappings should work.

**Desktop (please complete the following information):**
 - OS: neovim installed in wsl2 ubuntu 22.04 and neovide installed in win11
 - Neovide Version v0.10.0-dev
 - Neovim Version v0.10.4

**Comment by xarthurx** [comment]
Additionall, the following mapping also doesn't work:

```
<C-/>
<C-=>
<C-->
```
They all work in `neovim-qt` so I guess something is wrong here.

OS: win11
Neovide Version v0.10.4
Neovim Version v0.9.0

**Comment by xarthurx** [comment]
https://github.com/neovide/neovide/issues/1646

It seems a lot of mappings with `ctrl` is not working here.

**Comment by fredizzimo** [comment]
Also see this https://github.com/neovide/neovide/issues/994. We updated the winit version, but something else needs to be done still I think.

**Comment by fredizzimo** [comment]
This should be fixed by #1899, it would be great if you can test that.
